pkg/machine/qemu/machine_unix.go

```diff
@@ -43,3 +43,17 @@ func extractTargetPath(paths []string) string {
 	}
 	return paths[0]
 }
+
+func sigKill(pid int) error {
+	return unix.Kill(pid, unix.SIGKILL)
+}
+
+func killIfFound(pid int) (int, error) {
```

Comment:
Bad name of function.  I would figure this would call sigKill if the process exists.  But it does not send the SIGKILL signal, but the NULL function.

sigNullIfFound?

Reply:
Changed to findProcess, as that's what this is doing.